superset/viz.py

```diff
@@ -1453,10 +1453,10 @@ def get_data(self, df: pd.DataFrame) -> VizData:
                 x_values = [value["x"] for value in series["values"]]
                 min_x = min(x_values + ([min_x] if min_x is not None else []))
                 max_x = max(x_values + ([max_x] if max_x is not None else []))
-
+                series_keys = series["key"] if isinstance(series["key"], (list, tuple)) else [series["key"]]
```

Comment:
_looks_ like they come in as tuples or lists (looking at `viz_type="line"`) and defaults to an empty list if None on line 1451. 

I don't want to change much as I'm not very familiar with the codebase yet so sorry this is minimal effort. Hopefully, at some point, I can come back and help maintain this